documentation/installation/frequently_asked_questions.md

```
@@ -0,0 +1,12 @@
+# Installation FAQ
+
+Frequently Asked Questions for the Installation Process.
+
+---
+
+## Table of Contents
+
+- [TBA](#TBA)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)